### ISSUE TITLE
Focus ring issue with transition

### ISSUE THREAD

**Issue by matthieudou** [tailwindlabs/tailwindcss#3171]
### Describe the problem:

Hi,
First of all, thank you! Tailwindcss is awesome.
I'm having a little issue with the new `ring` utilities and I have no idea how it could be fixed.
If you apply some focus styles on a button for example with a transition there is a black box that's starting the animation and that is animated to the focus ring.

### Link to a minimal reproduction:
https://play.tailwindcss.com/oFbrZjD5an

**Comment by adamwathan** [comment]
This seems to be happening because you are using `transition-all`, which is transitioning other properties you don't want to transition (like the `outline` property). Switching to just `transition` fixes it for me:

https://play.tailwindcss.com/wK5X5VS8IP

The black box is definitely coming from the `outline` property being transitioned, so as long as you don't transition that property you're good!
